sql/2019/06_Fonts/06_09.sql

```diff
@@ -0,0 +1,32 @@
+#standardSQL
+
+-- counts the number of pages given number of font-face
+
+WITH
+  response_bodies AS (
```

Comment:
For counting the number of font resources per page, let's use the `type=font` field of the `summary_requests` table:

```sql
#standardSQL
SELECT
  APPROX_QUANTILES(fonts_per_page, 1000)[OFFSET(250)] AS p25_fonts_per_page,
  APPROX_QUANTILES(fonts_per_page, 1000)[OFFSET(500)] AS median_fonts_per_page,
  APPROX_QUANTILES(fonts_per_page, 1000)[OFFSET(750)] AS p75_fonts_per_page
FROM (
  SELECT
    COUNT(0) AS fonts_per_page
  FROM
    `httparchive.summary_requests.2019_07_01_*`
  WHERE
    type = 'font'
  GROUP BY
    pageid)
```

I'm also aggregating the distribution into 25,50,75th percentiles (2/4/6 fonts per page accordingly)

In general, avoid the response_bodies table if you can help it.